spec/poros/api_bill_spec.rb

```diff
@@ -2,12 +2,13 @@
 
 RSpec.describe ApiBill do
   describe "existance" do
-    it "exists and has attributes", :vcr do
+    xit "exists and has attributes", :vcr do
       params = {
```

Comment:
curious why we're "x"ing out these tests when we're merging it to main? I believe these tests were working before, but i see a bunch of prys in here, did something break? if so, we should try to fix before we merge to main